colorpair.py

```diff
@@ -0,0 +1,30 @@
+class ColorPair:

```

Comment:
Why does this need to be a class? Asking because it does not have any state...
In python, the file (colorpair.py) itself is a module with own namespace (doesn't interfere when importing along with other modules which may have tokens with same name). So a class is not needed to isolate the namespace.